contracts/src/arbitration/dispute-kits/DisputeKitClassic.sol

```diff
@@ -230,11 +232,13 @@ contract DisputeKitClassic is BaseDisputeKit, IEvidence {
 
         (uint96 subcourtID, , , , ) = core.disputes(_coreDisputeID);
         bytes32 key = bytes32(uint256(subcourtID)); // Get the ID of the tree.
-        uint256 drawnNumber = getRandomNumber();
 
         (uint256 K, uint256 nodesLength, ) = core.getSortitionSumTree(key, 0);
         uint256 treeIndex = 0;
-        uint256 currentDrawnNumber = drawnNumber % core.getSortitionSumTreeNode(key, 0);
+        uint256 currentDrawnNumber = uint256(
+            keccak256(abi.encodePacked(randomNumber, _coreDisputeID, round.votes.length))
```

Comment:
Equivalent of this in v1: https://github.com/kleros/kleros/blob/master/contracts/kleros/KlerosLiquid.sol#L493